x-pack/plugin/sql/qa/src/main/resources/agg-ordering.csv-spec

```diff
@@ -0,0 +1,26 @@
+aggSumWithColumnRepeatedWithOrderAsc
```

Comment:
This and the following test are ordering on the gender which is already tested in the aggs.csv-spec.
They should be changed to order on the aggregate function (SUM).

Reply:
changed